src/makeRobot.js

```diff
@@ -38,7 +38,95 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    x: 0,
+    y: 0,
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
it is possible to simplify it
```suggestion
    name,
    wheels,
    version,
```